apps/src/gamelab/GameLabSprite.js

```diff
@@ -453,7 +453,7 @@ var AABBops = function (p5Inst, type, target, callback) {
             if(displacement.y > 0)
               this.touching.top = true;
 
-            if(callback != undefined && typeof callback == "function")
+            if(callback !== undefined && typeof callback === "function")
```

Comment:
Isn't the first part of this actually just redundant? typeof callback == "function" implies callback != undefined